server/src/main/java/org/opensearch/cluster/routing/remote/InternalRemoteRoutingTableService.java

```diff
@@ -300,4 +300,15 @@ protected void doStart() {
     @Override
     protected void doStop() {}
 
+    @Override
+    public void deleteStaleIndexRoutingPaths(List<String> stalePaths) throws IOException {
+        try {
+            logger.debug(() -> "Deleting stale index routing files from remote - " + stalePaths);
+            blobStoreRepository.blobStore().blobContainer(BlobPath.cleanPath()).deleteBlobsIgnoringIfNotExists(stalePaths);
+        } catch (IOException e) {
+            logger.error("Error while deleting stale index routing files", e);
```

Comment:
Add details of paths which failed

Reply:
Exception will have details of stale paths which have failed to delete.

Reply:
Should we use () -> {} for error logs as well?